test/Mono.Linker.Tests.Cases/DataFlow/AnnotatedMembersAccessedViaReflection.cs

```diff
@@ -264,13 +264,22 @@ public static void Test ()
 		class AnnotatedMethodReturnValue
 		{
 			[return: DynamicallyAccessedMembers (DynamicallyAccessedMemberTypes.PublicMethods)]
-			public static Type StaticMethodWithAnnotatedReturnValue () => null;
+			public static Type StaticMethodWithAnnotatedReturnValue ()
+			{
+				return null;
+			}
```

Comment:
I don't really mind, but why did you change this?

Reply:
At some point the analyzer was failing on arrow expressions, now its fixed so Im getting back the arrows